CHANGELOG.md

```diff
@@ -0,0 +1,220 @@
+
+### Changed
+
+ - Moved much of the logic in spiceinit into a new library called [ALE](https://github.com/USGS-Astrogeology/ale). [#2370](https://github.com/USGS-Astrogeology/ISIS3/issues/2370)
+ - lronaccal no longer requires SPICE data available locally if it has been attached to the input cube with spiceinit. [#1790](https://github.com/USGS-Astrogeology/ISIS3/issues/1790)
+
+### Fixed
+
+ - qview no longer crashes when a band has only one pixel value in it. [#3323](https://github.com/USGS-Astrogeology/ISIS3/issues/3323)
+ - photomet now correctly reads array valued PvlKeywords. [#3608](https://github.com/USGS-Astrogeology/ISIS3/issues/3608)
+
+## [3.9.1] - 2019-11-19
+
+### Changed
+
+ - Improved hijitreg's documentation.
+ - Improved camdev's documentation.
+ - voy2isis is less picky about the exact instrument name.
+ - photomet now raises a warning when parameters are missing.
+ - makeflat and cisscal now use ISIS variables in their report files intead of absolute paths.
+
+### Fixed
+
+ - qmos now properly handles both PositiveWest and PositiveEast longitude domains.
+ - jigsaw's bundleout.txt output file now has proper alignment in its tables.
+ - cam2map no longer runs indefinitely with certain parameters.
+ - marci2isis no longer writes out a cube when it fails.
+ - marcical now uses the correct variable exposure time from the image label instead of a constant exposure time.
+
+## [3.9.0] - 2019-09-27
+
+### Added
+
+ - Users can now specify the output bit type for hideal2pds.
+ - oblique resolution is now available in qview's advanced tracking tool.
+ - oblique resolution statistics are now computed in caminfo.
+ - PDS4 exported image labels now have more precise ground ranges.
+ - PDS4 exported TGO CaSSIS image labels now contain additional fields.
+ - kerneldbgen can now take an explicit list of kernel files instead of a directory and a filter.
+ - FISH shell is now nominally supported.
+ - kerneldbgen now works with kernels that have an extremely large number of intervals.
+
+### Changed
+
+ - cisscal now matches version 3.9.1 of the IDL program.
+ - cisscal also now reports the matching IDL version.
+ - Improved the format of jigsaw's bundleout.txt output file.
+ - cam2map now uses better buffer sizes.
+ - voycal now reports all missing coefficients instead of just the first one encountered.
+
+## [3.8.1] - 2019-08-16
+
+### Changed
+
+ - Dawn FC's error message no longer tells users to bother Jeff Anderson.
+
+### Fixed
+
+ - caminfo now properly errors when passed a projected cube.
+ - Filenames with months in them are now properly translated using the host system's local instead of US English.
+ - findfeatures now properly resets the input images between algorithm runs when running in multi-algorithm mode.
+ - qmos no longer hangs when drawing a grid over a projected image with positive west longitude domain.
+
+## [3.8.0] - 2019-07-29
+
+### Changed
+
+ - Removed some dev tools form the installation environment.
```

Comment:
I'm sure this came from the original astrodiscuss post, but can you update form -> from?